doc/users_guide.rst

```diff
@@ -0,0 +1,306 @@
+	],
+
+	"host"        : '192.168.0.1',
+	"username"    : 'root',
+	"password"    : 'root',
+
+	"rtapp-calib" : {
+	    '0': 361, '1': 138, '2': 138, '3': 352, '4': 360, '5': 353
+	}
+    }
+
+    te = TestEnv(target_conf)
+
+We now have a dedicated class for the ``target_conf``, see :class:`lisa.env.TargetConf`.
+The most notable changes are as follows (see the doc for details):
+
+* ``"platform"`` is now ``"kind"``
+* ``"board"`` used to load some target-specific settings, which we got rid of.
+  The closest thing to it is ``"name"`` which is just a pretty-printing name and
+  has no extra impact.
+* You don't have to specify devlib modules to load anymore. All (loadable)
+  modules are now loaded. If you find some module too slow to load, you can
+  specify a list of modules to exclude.
+* LISA used to have ``target.config`` JSON file at its root. Its equivalent is
+  now ``target_conf.yml``, which is in YAML.
+
+.. admonition:: Cool new feature
+
+   :class:`lisa.env.TestEnv` instances can now be easily be  created
+   :meth:`via CLI<lisa.env.TestEnv.from_cli>`!
+
+
+Trace
++++++
+
+The :class:`lisa.trace.Trace` class hasn't changed much in terms of functionality,
+but we did rename/move things to make them more coherent.
+
+* Removed last occurences of camelCase
+* Removed big.LITTLE assumptions and made the code only rely on CPU capacities or
+  frequency domains, where relevant.
+* ``Trace.data_frame`` is gone:
+
+**LISA legacy**::
+
+  trace.data_frame.trace_event("sched_switch")
+
+**LISA next**::
+
+  trace.df_events("sched_switch")
+
+
+Analysis
+++++++++
+
+Analysis modules must now be called explicitely:
+
+**LISA legacy**::
+
+  trace.data_frame.latency_df("test")
+
+**LISA next**::
+
+  trace.analysis.latency.df_latency("test")
+
+.. admonition:: Cool new feature
+
+  Trace events required by the analysis methods are now automatically documented,
+  see :meth:`~lisa.analysis.frequency.FrequencyAnalysis.df_cpu_frequency_residency`
+  for instance.
+
+wlgen
++++++
+
+The :class:`lisa.wlgen.rta.RTA` class has been simplified somewhat:
+
+* :class:`lisa.wlgen.rta.RTATask` no longer has a superfluous ``get()`` method
+* ``RTA.conf()`` has been squashed inside alternative constructors, see
+  :meth:`lisa.wlgen.rta.RTA.by_str` and :meth:`lisa.wlgen.rta.RTA.by_profile`.
+
+**LISA legacy**::
+
+  profile = {}
+  profile["my_task"] = Periodic(duty_cycle_pct=30).get()
+
+  wload = RTA(te.target, "foo", calibration)
+  wload.conf(kind='profile', params=profile)
+
+**LISA next**::
+
+  profile = {}
+  profile["my_task"] = Periodic(duty_cycle_pct=30)
+
+  wload = RTA.by_profile(te, "foo", profile, res_dir, calibration)
+
+Kernel tests
+++++++++++++
+
+The ``Executor`` from LISA legacy has been entirely removed, and a new test
+framework has been put in place. Tests are now coded as straight Python classes,
+which means they can be imported and executed in scripts/notebooks without any
+additionnal effort. See :ref:`kernel-testing-page`.
```

Comment:
Should we mention bisector somewhere in this section ?

Reply:
I have another PR that adds some bits to the kernel testing page about how to run tests and all. Since I point it here we should be good.

Reply:
I'm under the process of streamlining integration between bisector and exekall, when it's done I'll write a howto page with a workflow from bisector to notebook visualization.